### ISSUE TITLE
[Request] New Command: /ae [tinker,enchanter,alchemist] <playerName> (Through Console)

### ISSUE THREAD

**Issue by EnderGamingFilms** [GC-spigot/AdvancedEnchantments#1143]
**Describe the Request**
So I want to use this plugin on my server but it seems it doesn't have a very essential aspect that I would assume it would have after being in development since 2017. This plugin does not have a way for console to open the respective guis for players.

The reason this is so important is that I want to use NPCs that the players have to right-click in order to open the gui. Technically, yes, you can setup the commands to use permissions in the config.yml and then use a CitizenCMD to give the player the permission when they try to open the gui, but it is just as easy to implement as it would be to just make a command that allows the console to just open the gui for the player without using third-party workaround (giving a permission to a player to open a gui then removes it I assume).

I have implemented this type of thing for my own plugin (to open a gui for the player through console), I've done this because I see developers doing this all the time, and this is pretty much the first plugin I have seen that heavily relies on guis but does not have this implements.

**TLDR;**
    Add the command <code>/ae [tinker,echanter,alchemist] \<playerName></code> that can be used by console so that the plugin owner can implement it in countless ways like, making a 1-time use item that opens the tinker gui, or not only be limited to using CitizenCMD to give the player the permission to open the gui (this is a plugin-specific feature to CitizenCMD and this effectively limited the compatibility of Advanced Enchantments).

**Comment by EnderGamingFilms** [comment]
**ALSO**
There is also a limitation with using CitizenCMD/CommandNPC, that clicking on the button to switch to the Alchemist or Tinkerer cannot be done since the player won't have the permissions to open them.

**Comment by GC-spigot** [comment]
Added
